### DISCUSSION TITLE
Is it a good idea to integrate nostr protocol into p2p layer?  If not, why?

### DISCUSSION THREAD

**Discussion by Yihen-Liu** [btcsuite/btcd#2088] [Q&A]

**Comment by GeorgeTsagk** [comment]
From the top of my head: the p2p layer is supposed to maintain the properties of a p2p network. Nostr would prove more efficient if you focused the communication to big relays, which is as good as having a central server delivering you the bitcoin peer gossip, instead of your peers.

**Comment by Roasbeef** [comment]
Moving to a discussion.
